index.html

```diff
@@ -169,11 +169,11 @@
       </section>
 
       <section>
-        <h2>The <dfn>Performance</dfn> interface</h2>
+        <h2>Extensions to the <dfn>Performance</dfn> interface</h2>
 
-        <p>The <a>Performance</a> interface hosts performance related attributes and methods used to retrieve the performance metric data from the <a>Performance Timeline</a>.</p>
+        <p>This extends the <a href='http://www.w3.org/TR/hr-time/#idl-def-Performance'>Performance</a> interface [[!HR-TIME-2]] and hosts performance related attributes and methods used to retrieve the performance metric data from the <a>Performance Timeline</a>.</p>
```

Comment:
done for hr-time-2 but further are blocked due to a bug in respec. I need to set some time aside to track it :(